src/components/include/utils/make_shared.h

```diff
@@ -34,6 +34,7 @@
 #define SRC_COMPONENTS_INCLUDE_UTILS_MAKE_SHARED_H_
 
 #include <new>
+#include <stddef.h>
```

Comment:
@AByzhynar Thank you for the notice and question. I think you are right and this change better be with <cstddef> header since the C++ header supposed to wrap the C names into std namespace. with C++ approach. So I will fix this.